pkg/cluster/storageSuffix.go

```diff
@@ -0,0 +1,33 @@
+package cluster
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	"github.com/Azure/ARO-RP/pkg/database"
+)
+
+func (m *manager) ensureStorageSuffix(ctx context.Context) error {
+	var err error
+
+	var f database.OpenShiftClusterDocumentMutator = func(doc *api.OpenShiftClusterDocument) error {
```

Comment:
Same comment than above for here. Though here I would say that it is really better to declare the function outside in its own func declaration. And then you can test it :)

Reply:
Tks. Done. Same comment as before.